static/hello_world.html

```diff
@@ -20,20 +21,17 @@
             if (xmlhttp.status == 200) {
                 document.getElementById("song_output").innerHTML = xmlhttp.responseText;
             }
-            else if (xmlhttp.status == 400) {
-                alert('dcfghnm');
-            }
             else {
-                alert('zxcvbnm, ' + xmlhttp.status);
+                alert('Error, ' + xmlhttp.status);
             }
           }
       };
 
       xmlhttp.setRequestHeader("Content-Type", "text/plain")
       xmlhttp.open("POST", "http://127.0.0.1:5000/format_chordpro/", true);
-      var inputHtml  = document.getElementById('song_input').value;
+      var inputHtml = document.getElementById('song_input').value;
       xmlhttp.send(inputHtml);
-      // document.getElementById('song_output').innerHTML=document.getElementById('song_input').value;
+      document.getElementById('song_output').innerHTML=document.getElementById('song_input').value;

```

Comment:
why have you uncommented it?

Reply:
you should remove this line

Reply:
It was an accident, but it doesn't work either way =(